lms/static/js/learner_dashboard/views/course_enroll_view.js

```diff
@@ -66,6 +66,12 @@
                         if (runKey){
                             this.model.updateRun(runKey);
                         }
+                        else{
+                            //Set back the unselected states
+                            this.model.setActiveRunMode(
+                                this.model.getUnselectedRunMode(
+                                    this.model.get('enrollable_run_modes')));
```

Comment:
Nitpick: This is pretty confusing, would it make sense to have a function on the model that can be called directly that achieves the same thing?